src/main/java/pl/antonina/tasks/util/LocationUtils.java

```diff
@@ -0,0 +1,17 @@
+package pl.antonina.tasks.util;
+
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+
+public class LocationUtils {
+
+    public static URI getLocation(long id) {
```

Comment:
In general it's good approach, but static methods are difficult to mock, so it's always worth to remember about this! :)